langwatch/prisma/schema.prisma

```diff
@@ -796,3 +800,18 @@ model Analytics {
     @@index([createdAt, key, projectId, numericValue])
     @@index([projectId])
 }
+
+model CustomRole {
+    id             String       @id @default(nanoid())
+    organizationId String
+    organization   Organization @relation(fields: [organizationId], references: [id])
+    name           String
+    description    String?
+    permissions    Json // Array of Permission strings
```

Comment:
@richhuth this seems like it might get us into trouble -- and also it says array but it's a json?